src/Storage/Storage.Management.Test/ScenarioTests/StorageAccountTests.cs

```diff
@@ -155,5 +155,12 @@ public void TestStorageAccountManagementPolicy()
         {
             TestRunner.RunTestScript("Test-StorageAccountManagementPolicy");
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestNewSetAzureStorageAccount_LargeFileShare()
+        {
+            TestRunner.RunTestScript("Test-NewSetAzureStorageAccount_LargeFileShare");
```

Comment:
Why using `_` here instead of using CamelCase？

Reply:
@wyunchi-ms 
Does it break any rule?
I don't prefer to change it, since it will take difficulty to merge changes back to storage branch.
Anyway, if it do break some name rules, I can change it.